### ISSUE TITLE
Would like to do this, but ...

### ISSUE THREAD

**Issue by sinanlag** [sebgl/htpc-download-box#55]
Hi,
I would very much like to do this, but as a first attempt, I'd rather not have the vpn yet. I'd do that later on as an enhancement. 

My problem is that I am very new to this. Would it be possible to get some hints/suggestions as to how I can implement this without a vpn?

Thx

**Comment by jbusuttil83** [comment]
It is not a problem to run this setup without vpn.
You need to comment out the vpn container from the docker-compose.yml file.

**Comment by emkowale** [comment]
Yeah...  I thought that way. I did what jbusuttil83 said, but I don't recommend it. My ISP shut me down after a few months. I was cheap and didn't get the VPN. Anyway, they slapped me on the wrist and turned me back on. I got NordVPN setup. It was only $89/2 years. I figure I've been overcharged for tv programming for decades. Big tech can go fly a kit. Plus with sebl's setup you don't even have to watch any media indoctrinating commercials! God Bless America!

**Comment by jbusuttil83** [comment]
@emkowale yes it would only be an acceptable solution if you do not want to use a VPN service with this service. If in your country ISPs block you from using these kind of things then you would have to use a VPN service.
